test/test_grudge.py

```diff
@@ -627,17 +628,17 @@ def f(x):
         op_global = op.nodal_sum(dcoll, dd, stiff - (stiff_t + kterm))
         op_local = op.elementwise_sum(dcoll, dd, stiff - (stiff_t + kterm + flux))
 
-        err_global = abs(op_global)
-        err_local = op.norm(dcoll, op_local, np.inf)
-        logger.info("errors: global %.5e local %.5e", err_global, err_local)
-
         # compute max element size
         from grudge.dt_utils import h_max_from_volume
 
-        h_max = h_max_from_volume(dcoll)
+        h_max = actx.to_numpy(h_max_from_volume(dcoll))
+        err_global = actx.to_numpy(abs(op_global))
+        err_local = actx.to_numpy(op.norm(dcoll, op_local, np.inf))
+        logger.info("errors: h_max %.5e global %.5e local %.5e",
+                h_max, err_global, err_local)
```

Comment:
This is unrelated, but the `actx.to_numpy` was done after the `logger.info` and the logging didn't appreciate getting a `cl.Array` to print, so I moved it around a bit.